lib/index.js

```diff
@@ -44,5 +44,8 @@ History.prototype.lengthForward = require('./prototype/length-forward')
 History.prototype.backward = require('./prototype/backward')
 History.prototype.forward = require('./prototype/forward')
 History.prototype.move = require('./prototype/move')
+History.prototype.forgetAllInDirection = require('./prototype/forget-all-in-direction')
```

Comment:
[It is private](https://github.com/PolicyStat/object-history/pull/9/files#diff-5f8ebb50eb0dbedd1ebad3dc174692e4R7). I did not follow the convention of prefixing `_` or any means of hiding the method.

The only thing I did is document it as private. Which means that Verb excludes it from the documentation.